test/performance/observed_concurency_test.go

```diff
@@ -199,7 +200,10 @@ func TestObservedConcurrency(t *testing.T) {
 		}
 	}
 
-	if err = testgrid.CreateTestgridXML(tc, tName); err != nil {
+	ts := junit.TestSuites{}
```

Comment:
I would consider adding a TODO here about re-adding the `CreateTestgridXML` function to `testgrid`. It really doesn't make sense repeating this pattern create-structture/add-suite/create-file again and again. @chaodaiG FYI